README.md

```diff
@@ -6,7 +6,7 @@
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
     <a href="CONTRIBUTING.md">Getting Involved</a>
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
-    <a href="https://gitter.im/open-telemetry/opentelemetry-service">Getting In Touch</a>
+    <a href="https://app.slack.com/client/T08PSQ7BQ/CJFCJHG4Q">Getting In Touch</a>
```

Comment:
This should probably be https://cloud-native.slack.com/archives/C01N6P7KR6W (#otel-collector) instead of #opentelemetry.

Reply:
Adding this as suggestion.